CHANGELOG.rst

```diff
@@ -2,6 +2,211 @@
+  if the telemetry is enabled. One can run it with the following command:
+     `agent diagnose show-metadata agent-telemetry`. See
+  `docs <https://docs.datadoghq.com/data_security/agent/#telemetry-collection>` for more details.
+
+- Convert Prometheus style Counters and Histograms used in Agent telemetry
+  from monotonically increasing to non-monotonic values (reset on each scrape).
+  In addition de-accumulate Prometheus Histogram bucket values on each scrape.
+
+- Added support for more than 100 Aurora clusters in a user's account when using database autodiscovery
+
+- Adds some information about the SNMP autodiscovery status in the Agent status.
+
+- Adds a dedicated CRI-O Workloadmeta collector, enabling metadata collection
+  for containers running on a CRI-O runtime.
+
+- Enables a cache for SQL and MongoDB obfuscation. This cache is enabled by default but can be disabled by setting `apm_config.obfuscation.cache.enabled` to `false`.
+
+- Improved logging to add visibility for latency and transport protocol
+
+- Add a new configuration option ``log_level`` for commands where the logger is disabled by default.
+
+- Adds initial Windows support for TCP probes in Network Path.
+
+- Query Aurora instances per cluster to allow up to 100 instances per cluster
+  rather than 100 instances total.
+
+- The AWS Lambda Extension is now  able to read the full 128-bit trace ID 
+  from the headers of the end-invocation HTTP request made by dd-trace or the
+  datadog-lambda-go library.
+
+- Standardized cluster check tagging across all environments, allowing DD_TAGS, DD_EXTRA_TAGS, DD_CLUSTER_CHECKS_EXTRA_TAGS, and DD_ORCHESTRATOR_EXPLORER_EXTRA_TAGS to apply to all cluster check data when operating on the Cluster Agent, Node Agent, or Cluster Checks Runner.
+
+
+.. _Release Notes_7.61.0_Deprecation Notes:
+
+Deprecation Notes
+-----------------
+
+- Deprecates the `apm_config.obfuscation.sql.cache` option in favor of `apm_config.obfuscation.cache`.
+
+- Remove deprecated config `otlp_config.metrics.instrumentation_library_metadata_as_tags`. Use `otlp_config.metrics.instrumentation_scope_metadata_as_tags` instead.
+
+- The remote tagger will attempt to connect to the core agent indefinitely until it is successful. 
+  The ``remote_tagger_timeout_seconds`` configuration is removed, and the timeout is no longer configurable.
+
+- The remote tagger for the trace-agent and security-agent is now always enabled and can not be disabled 
+  ``apm_config.remote_tagger``,  ``security_agent.remote_tagger``, and ``event_monitoring_config.remote_tagger`` config entries are removed.
+
+
+.. _Release Notes_7.61.0_Security Notes:
+
+Security Notes
+--------------
+
+- Fix CVE-2025-21613
+
+- Update ``golang.org/x/crypto`` to fix CVE-2024-45337.
+
+
+.. _Release Notes_7.61.0_Bug Fixes:
+
+Bug Fixes
+---------
+
+- Cluster Agent: Don't overwrite the LD_PRELOAD environment variable if it's already set, append the path to Datadog's injection library instead.
```

Comment:
Should this entry be moved to `CHANGELOG-DCA.rst` instead ?

Reply:
Good catch, fixed!